[CrateDB] Add support for data acquisition and data export (#148)
daq-tools/kotori: doc/source/about/technologies.rst

- Kotori_, a data acquisition, graphing and telemetry toolkit

> - Grafana_, a graph and dashboard builder for visualizing time series metrics

Author: ```suggestion
- Grafana_, a graph and dashboard builder for visualizing time-series metrics
```

> - CrateDB_, a time-series database ¹

Author: ```suggestion
- CrateDB_, a time-series database with document features and more ¹
```

- InfluxDB_, a time-series database ¹
- Mosquitto_, an MQTT message broker
- MongoDB_, a document store (optional) ²³

> | ¹ Kotori can either use CrateDB or InfluxDB as timeseries database.

Author: ```suggestion
| ¹ Kotori can either use CrateDB or InfluxDB as time-series database.
```
